activity_calendar/templates/activity_calendar/googlecalendar.html

```diff
@@ -3,12 +3,15 @@
 {% load static %}
```

Comment:
Out of scope for this PR, but this view can definitely be removed.

Reply:
It's been removed anyway